samples/LowAllocationWebServer/LowAllocationWebServerLibrary/OwnedBuffer.cs

```diff
@@ -105,13 +109,35 @@ public void Advance(int bytes)
 
         protected override void Dispose(bool disposing)
         {
-            var array = Array;
+            var array = _array;
             base.Dispose(disposing);
             Free(array);
             if (_next != null) {
                 _next.Dispose();
             }
             _next = null;
         }
+
+        public override Span<byte> GetSpan(int index, int length)
```

Comment:
Why do we need this if we already expose span?
Why is this throwing on dispose and span does not?